ui/components/app/transaction-decoding/components/ui/copy-raw-data/copy-raw-data.component.js

```diff
@@ -0,0 +1,38 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '../../../../../ui/tooltip/tooltip';
+import CopyIcon from '../../../../../ui/icon/copy-icon.component';
+import { I18nContext } from '../../../../../../contexts/i18n';
+
+import { useCopyToClipboard } from '../../../../../../hooks/useCopyToClipboard';
+
+const CopyRawData = ({ data }) => {
+  const t = useContext(I18nContext);
+  const [copied, handleCopy] = useCopyToClipboard();
+
+  return (
+    <div className="copy-raw-data">
+      <Tooltip position="right" title={copied ? 'Copied!' : ''}>
```

Comment:
We should use `copiedExclamation` localization here

Reply:
1efaa13e6